tsconfig.json

```diff
@@ -7,5 +7,7 @@
   "compilerOptions": {
     // output to dist/ dir
     "outDir": "dist/",
+    "jsx": "react-jsx",
```

Comment:
so this is not compatible with older React versions, which is why I have yet to use it and why I still have `import React` (among other things). you might notice that in the `peerDeps` here, `react-signature-canvas` is backward compatible with at least React v14.

this would require changing the `peerDeps` version. it looks like the React team did [backport this all the way to React v14](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-a-jsx-transform), but the range would need to be changed to only those versions and above.